readme.md

```diff
@@ -47,6 +47,7 @@ Please raise a [Pull Request](https://github.com/wizztjh/awesome-webvr/pulls) to
 
 ### Demos
 
+- [ArtSocks](https://art-socks.herokuapp.com/demo.html) - An experiment with VR and web sockets by [dziemid](https://twitter.com/dziemid) (Google Cardboard)
```

Comment:
Exactly, webvr boilerplate. Good to hear it works on oculus, I've only had a chance to test on cardboard :-) - For context - here is short blog post about the demo gregdmd.com/blog/2015/11/24/campjs-project-artsocks-vr